composer.json

```diff
@@ -1,5 +1,16 @@
 {
+    "name": "lndrest/composer",
+    "description": "An lnd class",
```

Comment:
we need a better description. 
maybe something like: 

"A client for the lnd REST API"